### ISSUE TITLE
Additional Optimizing Metrics for Binary Classification: Precision or Recall

### ISSUE THREAD

**Issue by judahr** [dotnet/machinelearning-modelbuilder#2797]
In binary classification, there are situations where only false positives or only false negatives are costly, we would want to tune the model on simply Precision or Recall.  This would better optimize models based on these scenarios.

In Advanced training options, under the optimizing metric include a Precision option and a Recall option.  Use this to tune the Model Builder for higher precision or higher recall.

**Comment by LittleLittleCloud** [comment]
@JakeRadMSFT @luisquintanilla

**Comment by luisquintanilla** [comment]
Makes sense to me. @LittleLittleCloud is it just a matter of adding those options in the UI or is work required in the AutoML API?

**Comment by LittleLittleCloud** [comment]
Just adding options on UI
